Taskfile.dist.yaml

```diff
@@ -35,3 +280,427 @@ tasks:
+
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault secrets enable -path=trdl vault-plugin-secrets-trdl
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault write trdl/configure s3_secret_access_key=minioadmin s3_access_key_id=minioadmin s3_bucket_name=trdl s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.minio_port}} required_number_of_verified_signatures_on_commit=0 git_repo_url=https://github.com/werf/trdl
+
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault secrets enable -path=kubedog vault-plugin-secrets-trdl
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault write kubedog/configure s3_secret_access_key=minioadmin s3_access_key_id=minioadmin s3_bucket_name=kubedog s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.minio_port}} required_number_of_verified_signatures_on_commit=0 git_repo_url=https://github.com/werf/kubedog
+
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault secrets enable -path=werf vault-plugin-secrets-trdl
+      - VAULT_TOKEN=root VAULT_ADDR=http://$(docker inspect {{.vault_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.vault_port}} vault write werf/configure s3_secret_access_key=minioadmin s3_access_key_id=minioadmin s3_bucket_name=werf s3_region=ru-central1 s3_endpoint=http://$(docker inspect {{.minio_container}} --format "{{ .NetworkSettings.IPAddress }}"):{{.minio_port}} required_number_of_verified_signatures_on_commit=0 git_repo_url=https://github.com/werf/werf git_trdl_channels_branch=multiwerf
+
+  setup:
+    desc: Set up MinIO and Vault development environment
+    dir: "{{.dir}}"
+    deps:
+      - build
+    cmds:
+      - task: setup:minio
+      - task: setup:vault
+      - touch .setup
+    sources:
+      - "cmd/vault-plugin-secrets-trdl"
+    generates:
+      - .setup
+    vars:
+      dir: '{{.dir | default "server"}}'
```

Comment:
Not gonna repeat myself anymore. Dir should be hardcoded in (almost?) all tasks.